docs/ibm-cloud-rules.md

```diff
@@ -2528,20 +2534,23 @@ paths:
 </table>
 
 
-### Rule: operation-summary
```

Comment:
Again, I expect this to be a new rule and not a replacement

Reply:
Yep, good catch.  It's fixed now.